Bouquet.h

```diff
@@ -8,33 +8,31 @@
 class Bouquet
 {
 private:
-    vector <Flowers> b;
-    int bouquetprice = 0;
+    vector <Flowers> bouquet;
+    int bouquet_price = 0;
 
 public:
     Bouquet();
 
     Bouquet(int number, vector<Flowers> assortment)
     {
-        for (int i = 0; i <number; i++)
+        for (int i = 0; i < number; i++)
         {
             int index = rand() % assortment.size();
-            b.push_back(assortment[index]);
-            bouquetprice += assortment[index].getprice();
-
+            bouquet.push_back(assortment[index]);
+            bouquet_price += assortment[index].getprice();
         }
-
-
     }
-    void databouquet()
+    ~Bouquet() {}
+   
+    void dataBouquet()
     {
-        for (int i = 0; i < b.size(); i++)
+        for (int i = 0; i < bouquet.size(); i++)
         {
-            cout << b[i].getname() << " " << b[i].getcolor() << " " << b[i].getsize() << " " << b[i].getprice() << endl;
+            cout << bouquet[i].getname() << " " << bouquet[i].getcolor() << " " << bouquet[i].getsize() << " " << bouquet[i].getprice() << endl;
         }
-        cout << "Price: " << bouquetprice << endl;
+        cout << "Price: " << bouquet_price << endl;
     }
     ~Bouquet() {}
```

Comment:
There is one destructor right after construcor. Please remove this one

Reply:
Fixed.